azure-pipelines.yml

```diff
@@ -135,18 +135,24 @@ stages:
        - script: git submodule update --init --recursive
          displayName: "Init Git Submodules"
          condition: succeeded()
-       - script: cmake -S . -B ./build/ -DCMAKE_PREFIX_PATH="C:\Qt\6.5.1\msvc2019_64"
+       - script: cmake -S . -B ./build/ -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH="C:\Qt\6.5.1\msvc2019_64"
```

Comment:
does it also work when you remove this flag?

Reply:
yeah